packages/cosmic-swingset/test/scenario2.js

```diff
@@ -10,13 +12,33 @@ export const pspawn = (bin, { spawn, cwd }) => {
     /** @type {import('child_process').ChildProcess} */
     let child;
     const exit = new Promise((resolve, reject) => {
-      // console.debug('spawn', bin, args, { cwd: makefileDir, ...opts });
-      child = spawn(bin, args, { cwd, ...opts });
-      child.addListener('exit', code => {
+      // When stderr is otherwise ignored, spy on it for inclusion in error messages.
+      /** @type {Promise<string> | undefined} */
+      let stderrP = Promise.resolve('');
+      // https://nodejs.org/docs/latest/api/child_process.html#optionsstdio
+      let { stdio = 'pipe' } = opts;
+      if (stdio === 'ignore' || stdio[2] === 'ignore') {
+        stderrP = undefined;
+        stdio = typeof stdio === 'string' ? [stdio, stdio, stdio] : [...stdio];
+        stdio[2] = 'pipe';
+      }
```

Comment:
This is just my style.  Feel free to ignore:

```suggestion
      /** @type {Promise<string> | undefined} */
      let stderrP;
      // https://nodejs.org/docs/latest/api/child_process.html#optionsstdio
      let { stdio = 'pipe' } = opts;
      if (stdio === 'ignore' || stdio[2] === 'ignore') {
        stdio = typeof stdio === 'string' ? [stdio, stdio, stdio] : [...stdio];
        stdio[2] = 'pipe';
      } else {
        stderrP = Promise.resolve('')
      }
```

Reply:
Thanks, that does read a bit better.